PWGUD/Tasks/upcForward.cxx

```diff
@@ -113,18 +124,23 @@ struct UPCForward {
     if (iMuonTracknumber != 2) {
       return;
     }
+
     registry.fill(HIST("hSelectionCounter"), 4);
     if (!ispositive || !isnegative) {
       return;
     }
     registry.fill(HIST("hSelectionCounter"), 5);
 
-    if (-4 < p1.Eta() < -2.5 || -4 < p2.Eta() < -2.5) {
-
+    if ((p1.Eta() < -4) || (p2.Eta() < -4)) {
+      return;
+    }
+    if ((p1.Eta() > -2.5) || (p2.Eta() > -2.5)) {
       return;
     }
```

Comment:
Here you apply hardcoded selections on the muons, but you already have implemented Filters which cut on this based on Configurables. So this is not needed.

Reply:
@iarsene I removed the selections.